src/components/Validate/index.tsx

```diff
@@ -1,7 +1,7 @@
 import Box from '@mui/material/Box'
 import { useNavigate } from 'react-router-dom'
 import Typography from '@mui/material/Typography'
-import { TextField, Button, MenuItem, FormControl, InputLabel, OutlinedInput, InputAdornment } from '@mui/material'
```

Comment:
why did you remove these i just added them?

Reply:
I replaced the FormControl with the text field